contrib/completion/bash/docker

```diff
@@ -157,6 +157,17 @@ __docker_networks() {
 	COMPREPLY=( $(compgen -W "$networks" -- "$cur") )
 }
 
+__docker_containers_in_network() {
+	local sed_name_command='s/^ \+"Name": "\(.*\)",\?/\1/p'
+	local sed_id_command='s/^ \+"\([0-9a-f]\{64\}\)":.*/\1/'
+	[ ${DOCKER_COMPLETION_SHOW_IMAGE_IDS:-none} != "none" ] && sed_id_command=${sed_id_command}p
+
+	local containers=$(__docker_q network inspect $1 \
```

Comment:
Does `network inspect` not have a `-f` flag for formatting output? :disappointed: